flytestdlib/.golangci.yml

```diff
@@ -26,7 +26,6 @@ linters:
     - structcheck
     - typecheck
     - unconvert
-    - unparam
```

Comment:
Disabling since I was getting 
```
cache/auto_refresh.go:433:2: `newAutoRefreshCacheWithClock` - `parallelizm` always receives `10` (unparam)
        parallelizm, size int, scope promutils.Scope, clock clock.WithTicker) (AutoRefresh, error) {
        ^
```

I could omit this from my specific line but people generally seem to think this linter is pretty annoying.